app/services/reporting/total_user_count_report.rb

```diff
@@ -10,14 +10,14 @@ def initialize(report_date = Time.zone.today)
 
     def total_user_count_report
       [
-        ['All-time user count'],
-        [total_user_count],
+        ['All-time user count', 'Total verified users', 'Total annual users'],
+        [total_user_count, verified_user_count, annual_total_user_count],
```

Comment:
I think a format like in this comment would be clearer?

https://github.com/18F/identity-idp/pull/9376#discussion_r1357472797

Reply:
Oh! I actually agree. Will change that.